edx_django_utils/monitoring/code_owner/middleware.py

```diff
@@ -51,6 +54,9 @@ def _set_code_owner_metric(self, request):
             return
         if not path_error:
             # module found, but mapping wasn't configured
+            code_owner = self._set_code_owner_metric_catch_all()
+            if code_owner:
+                set_custom_metric('code_owner', code_owner)
```

Comment:
**Reviewer Note**: Although this code would be less noisy if `_set_code_owner_metric_catch_all()` set the `code_owner` metric, I went with this option so that this method wouldn't be setting `code_owner` for some cases (the existing code), and not for other cases (the new code).